test/test-util.js

```diff
@@ -121,8 +121,13 @@ function testStateTransitions(ns, initial, newState, finalState, incNoDelta, sta
 function prepareCluster(insert_fns) {
     return function(t, tc, n) {
         return [
-            dsl.assertDetectChecksumMethod(t, tc),
+            // By waiting for the first ping we make sure the SUT is ready to go.
+            // We don't consume it so other tests (especially the ping-tests)
+            // can still assert implementation details related to ping distribution.
+            dsl.waitForPing(t, tc, false),
```

Comment:
The idea behind it is to wait until the node starts pinging. At that point we can consider it to be fully ready and start running the other validations.